src/mz390.java

```diff
@@ -12042,61 +12304,104 @@ private void exec_pc_d2a(){
     	 * convert decimal string to int
     	 */
     	check_setc_quotes(1); // RPI 1139
-    	boolean save_opt_allow = tz390.opt_allow; // RPI 1204
-    	tz390.opt_allow = true;           // RPI 1204
-    	seta_value = get_seta_stack_value(-1);
-    	tz390.opt_allow = save_opt_allow; // RPI 1204
 		setc_value1 = get_setc_stack_value();
+		if (setc_value1.length() == 0) {                                     // #509
+			// HLASM LR states this case indicates an error.                 // #509
```

Comment:
I encountered the LR abbreviation in many of your comments. First time I saw it, it took me a while before I realized it was not a LR instruction, but a reference to the Language Reference. Have you defined this abbreviation in each program where you use it? Maybe I'm being over-sensitive. But would prefer LangRef or something like that rather than LR. Just a thought - what do you think?

Reply:
I think I used "HLASM Language Reference" for the first occurrence and "HLASM LR" for all remaining occurrences. I can see where someone jumping in the middle might need what you suggested. Will make the updates.